helpers/terraform/variables.tf

```diff
@@ -39,7 +39,7 @@ variable "additional_zones" {
 
 variable "machine_type" {
   description = "Machine type for the kubernetes nodes"
-  default     = "n1-standard-8"
+  default     = "custom-10-30720"
```

Comment:
The fact that this increase is causing issues for our testing environment might be a sign that this should be announced as a breaking chance in the release notes. At least breaking for anybody using the current default resource limits.

Reply:
In our testing environment we are deploying 21 Elasticsearch pods (7 scenarios x 3 nodes) per clusters and we can assume that the pod are never requiring more resources than set in requests resources during the tests.

This change cause resource needed for Elasticsearch pods grow from 2.1 CPU to 21 CPU in our testing environment.

IMHO standard usage is closer to 3 long running Elasticsearch pods per cluster which are consuming CPU and memory limits, so I'm not sure that the impact should be considered as a breaking change. WDYT?

Reply:
The impact was big enough to cause us issues in our testing environment because of resource capacity. A 10x increase in the amount of CPUs requested by default could also cause issues for others too. For clusters with autoscaling enabled they might suddenly end up with 10 times the amount of Kubernetes node running after a version bump.

I don't think the change is big enough to avoid doing it, but I do think it would be nice to give all users a heads up including documentation for how to keep the existing configuration.

Reply:
That sounds good to me, I'll merge the PR and will add a note in next release's changelog about this change and how to keep existing config.

Reply:
@Crazybus notice added in https://github.com/elastic/helm-charts/pull/476/commits/293f0ec38fa00a74fd80c0beee3a67fc2c886f0e

Reply:
Perfect thank you!